### PR TITLE
Various updates to remote-config

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

Catch up on remote-config

### CODE REVIEW

**Comment by rfay** [comment]
Any suggestions on content changes or additional items are welcome, thanks!

File: remote-config.jsonc [hunk 1]
```diff
@@ -90,13 +90,13 @@
           "message": "`ddev snapshot -a` will back up all your project databases."
         },
         {
-          "message": "You're invited to the DDEV Advisory Group, bimonthly on first Wednesday"
+          "message": "You're invited to the DDEV Advisory Group, bimonthly on first Wednesday, see the schedule and sign up at https://www.meetup.com/ddev-events/"
```
Comment (tyler36) [inline remote-config.jsonc]:
```suggestion
          "message": "You're invited to the DDEV Advisory Group, first Wednesday every two months, see the schedule and sign up at https://www.meetup.com/ddev-events/"
```

I always get confused with "bimonthly". Happy to go with "every odd-numbered month", "every other month".

File: remote-config.jsonc [hunk 2]
```diff
@@ -194,47 +182,47 @@
         {
           "message": "A key goal for DDEV in 2024 is to make it possible for maintainer Stas Zhuk to work full-time on DDEV, see https://ddev.com/blog/lets-fund-stas-maintainer/"
         },
```
Comment (tyler36) [inline remote-config.jsonc]:
```suggestion
        {
          "message": "A key goal for DDEV in 2025 is to make it possible for maintainer Stas Zhuk to work full-time on DDEV, see https://ddev.com/blog/lets-fund-stas-maintainer/"
        },
```

Bump year to 2025, assuming this is still a goal.

**Review by tyler36 — CHANGES_REQUESTED** [review]

File: remote-config.jsonc [hunk 3]
```diff
@@ -174,46 +168,46 @@
...
-          "message": "DDEV 2024 plans: https://ddev.com/blog/2024-plans"
+          "message": "DDEV v1.24 changed some defaults for new projects, using PHP 8.3 and Node.JS 22."
         },
         {
-          "message": "Did you know that DDEV has experimental support for Python projects like Django? We'd love to have your feedback!"
+          "message": "DDEV Swag is available, including t-shirts, mugs, etc. see https://ddev.threadless.com/ - thanks @bmartinez287!"
         },
         {
-          "message": "DDEV v1.23 changed some defaults for new projects, using PHP 8.2, MariaDB 10.11, and Node.JS 20. ddev-webserver is now based on Debian 12 Bookworm."
+          "message": "DDEV allows you to rapidly profile the performance of your PHP code using XHProf with `ddev xhprof on`. Try out https://github.com/ddev/ddev-xhgui !"
```
Comment (stasadev) [inline remote-config.jsonc:177]: We have a similar entry, please remove one of them:

```
"Want to profile your code in an improved more beautiful manner with Xhprof? Try out @tyler36's https://github.com/ddev/ddev-xhgui"
```

File: remote-config.jsonc [hunk 4]
```diff
@@ -174,46 +168,46 @@
...
-          "message": "It's the DDEV community that makes DDEV what it is. THANK YOU!"
+          "message": "Use 'ddev_version_constraint' in your .ddev/config.yaml to make sure all your team members are on the same minimum DDEV version, for example `ddev_version_constraint: '>=v1.24.0'`"
         },
         {
-          "message": "The DDEV Foundation is now an official US 501(c)(3) organization. That means the foundation doesn't have to pay taxes and any financial contribution to it is tax-deductible. https://ddev.com/blog/501c3/"
+          "message": "Explicit support for the `symfony` project type was added in DDEV v1.24.0, so now you can `ddev config --project-type=symfony`."
         },
         {
-          "message": "Have you ever wondered how 'ddev.site' domain names work? Read all about it at https://ddev.com/blog/ddev-name-resolution-wildcards"
+          "message": "Want to know more about using vite with DDEV? The Vite blog is superbly maintained by Matthias Andrasch with the latest information: https://ddev.com/blog/working-with-vite-in-ddev/"
```
Comment (stasadev) [inline remote-config.jsonc:201]: We have a similar entry, please remove one of them:

```
"Read about Vite in DDEV, thanks to Matthias Andrasch: https://ddev.com/blog/working-with-vite-in-ddev/"
```

File: remote-config.jsonc [hunk 5]
```diff
@@ -279,6 +267,7 @@
         {
           "message": "DDEV already supports PHP 8.4!"
         },
+
```
Comment (stasadev) [inline remote-config.jsonc]:
```suggestion
```

**Review by stasadev — CHANGES_REQUESTED** [review]
I noticed old Discord link, and old mailpit link:

```diff
-https://discord.gg/hCZFfAMc5k
+https://discord.gg/5wjP76mBJD
```

```diff
-ddev launch -m
+ddev mailpit
```

**Review by stasadev — APPROVED** [review]

**Pull request merged**